### PR TITLE
[lldpmgr] add mgmt ip to lldpd conf template, handle port description/alias changes

### PR DESCRIPTION
**Pull request by mykolaf**
Signed-off-by: Mykola Faryma <mykolaf@mellanox.com>

**- What I did**
Add mgmt ip to lldpd.conf.j2
Change lldpmgrd to handle port description configured in CONFIG_DB and config the lldpd accordingly, instead of engineering some description which is never synced with DB. 
**- How I did it**
 - remove the description generating logic
 - add logic to listen to CONFIG_DB PORT table canges

**- How to verify it**
 sonic-cfggen -m /etc/sonic/minigraph.xml -p  /usr/share/sonic/device/x86_64-mlnx_msn2100-r0/ACS-MSN2100/port_config.ini -t lldpd.conf.j2
Result:
```
configure ports eth0 lldp portidsubtype local eth0
configure system ip management pattern 10.210.10.21
```

Config some port descriptions/alias, check lldp local port info

**- Description for the changelog**

Update lldpmgrd logic

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: dockers/docker-lldp-sv2/lldpmgrd [hunk 1]
```diff
@@ -135,34 +134,21 @@ class LldpManager(object):
...
-                    log_info("Failed to retrieve neighbor host name for port '{}'. Not adding port description.".format(port_name))
-
-                if not neighbor_portname:
-                    log_info("Failed to retrieve neighbor port name for port '{}'. Not adding port description.".format(port_name))
+        # if there is a description available, also configure that
+        if port_desc:
+            lldpcli_cmd += " description {}".format(port_desc)
         else:
-            log_info("Unable to retrieve neighbor information for port '{}'. Not adding port description.".format(port_name))
+            log_info("Unable to retrieve description for port '{}'. Not adding port description")
```
Comment (jleveque) [inline dockers/docker-lldp-sv2/lldpmgrd]: Why remove the string formatting argument? Now there is a formating specifier without a matching argument.

Comment (mykolaf) [reply dockers/docker-lldp-sv2/lldpmgrd]: Hi @jleveque , can you approve?

**Comment by liat-grozovik** [comment]
@jleveque your comments were addressed. can you please review and approve?

**Comment by yxieca** [comment]
Made to 201811 branch on 2/14/2019

**Pull request merged**